tests/system/python/packages/test_statistics_history_notification_rule.py

```diff
@@ -167,10 +167,10 @@ def test_stats_readings_south(self, clean_setup_fledge_packages, reset_fledge, s
         for audit_detail in resp1['audit']:
             if "test #1" == audit_detail['details']['name']:
                 assert "NTFSN" == audit_detail['source']
-        # Waiting for 60 sec to get 2 more NTFSN entries if rule is triggered properly
-        time.sleep(60)
+        # Waiting for 90 sec to get 2 more NTFSN entries if rule is triggered properly
```

Comment:
What is the trigger frequency?

Reply:
30 seconds

Reply:
So should not we keep it 70 to 80 instead of 90?